src/main/archetype/ui.apps/pom.xml

```diff
@@ -55,6 +55,7 @@
                     <group>${groupId}</group>
                     <name>${artifactId}</name>
                     <packageType>application</packageType>
+                    <skipSubPackageValidation>true</skipSubPackageValidation>
```

Comment:
why are we skipping package validation?

Reply:
We have wknd and easel themes in the hierarchy which are zips. This plugin thinks that they are packages and fail the validation. skipSubPackageValidation solves that.
I tried other methods which can possibly skip validation for our theme zips but didn't find them.

Reply:
done